saleor/product/migrations/0149_merge_20210927_0941.py

```diff
@@ -0,0 +1,13 @@
+# Generated by Django 3.2.7 on 2021-09-27 09:41
+
+from django.db import migrations
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ("product", "0148_merge_0147_auto_20210623_1304_0147_auto_20210817_1015"),
+        ("product", "0148_producttype_type"),
```

Comment:
Can we re-generate migrations for this PR from scratch, maybe we can get rid of these merge migrations this way?

Reply:
With pleasure :)